app/scripts/directives/istagSelect.js

```diff
@@ -21,31 +25,102 @@ angular.module("openshiftConsole")
       restrict: 'E',
       scope: {
         istag: '=model',
-        selectDisabled: '='
+        selectDisabled: '=',
+        includeOpenshift: '=',
+        customTagValue: '='
       },
       templateUrl: 'views/directives/istag-select.html',
-      link: function($scope){
+      controller: function($scope){
         $scope.isByNamespace = {};
         $scope.isNamesByNamespace = {};
 
+        // Check if the istag object contains data about namespace/imageStream/tag so the ui-select will be pre-populated with them
+        $scope.prepopulate = ($scope.istag.namespace && $scope.istag.imageStream && $scope.istag.tagObject.tag) ? true : false ;
```

Comment:
Shouldn't need to be a scope var since it's not used in the view. Although you'd need to give it a different name since you have a function with that name.

```js
var prepopulate = $scope.istag.namespace && $scope.istag.imageStream && scope.istag.tagObject.tag;
```